### PR TITLE
[0.11.0] Migration: `next-translate` in Account

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
## Code Changes
- Migrated `account.json` to `next-translate`
  Resolves FRO-163 [(link)](https://linear.app/skiso/issue/FRO-163/migrate-account-pages)

## Preview
Preview available at https://pr3.mysk.school/.

### CODE REVIEW

**Review by Sun-52 — CHANGES_REQUESTED** [review]
The translation of the report issue button without children and the report issue dialog don't work in account pages.

**Comment by SiravitPhokeed** [comment]
@Nubboyy What component?

@Sun-52 The Report Issue Dialog component is simply not migrated yet. Do you think that migration should be done in this PR now? I wanted to keep these PRs small so you’re comfortable reading them in one sitting, but I can migrate it now if you think I should.

**Comment by Sun-52** [comment]
@SiravitPhokeed I was only thinking that everything on the about page should be working and I forgot to think that other parts would be migrated in later PR. Sorry.

**Comment by SiravitPhokeed** [comment]
> I was only thinking that everything on the about page should be working

@Sun-52 You’re right to do that. Please do continue to be vigilant for any broken parts because they may well not be intentional.

File: translations/account/certificates/receivingOrderDialog/th.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "title": "ลำดับการรับที่ {{order, number}}",
+  "desc": "นักเรียนเป็นคนที่ {{order}} ที่ได้รับประกาศเกียรติคุณ โปรดจำตัวเลขนี้เพราะโต๊ะลงทะเบียนจะแบ่งตามลำดับนี้",
```
[the remark this answers is not in the document]
Comment (SiravitPhokeed) [reply translations/account/certificates/receivingOrderDialog/th.json:3]: `number` is not a variable. It‘s a [formatter](https://www.i18next.com/translation-function/formatting#number). Line 3 you cited is missing the formatter, which is incorrect.

**Comment by SiravitPhokeed** [comment]
@Nubboyy @Sun-52 Sorry about that! Have another look.

**Pull request merged**